.github/workflows/dependent-issues.yml

```diff
@@ -0,0 +1,36 @@
+---
+name: Dependent Issues
+
+on:
+  issues:
+    types:
+      - opened
+      - reopened
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+  pull_request:
+    types:
+      - opened
+      - reopened
+  schedule:
+    - cron: '0 * * * *' # check hourly
+
+jobs:
+  check:
+    name: check-dependent-PRs
```

Comment:
These names should be written for display on the GH UI, with spaces and such. The job name will be listed under the workflow name, so they should build on each other.